src/logger.ts

```diff
@@ -1,3 +1,45 @@
+// tslint:disable-next-line:no-implicit-dependencies
+import * as core from 'express-serve-static-core';
 import * as pino from 'pino';
 
 export const logger = pino();
+
+/**
+ * log middleware
+ */
+export function logMiddleware(): core.RequestHandler {
+    const handler = (req: any, res: any, next: core.NextFunction) => {
+        const startTime = Date.now();
+        function writeLog(): void {
+            const responseTime = Date.now() - startTime;
+            res.removeListener('finish', writeLog);
+            res.removeListener('close', writeLog);
+            const logMsg = {
+                req: {
+                    url: req.originalUrl.split('?')[0],
+                    method: req.method,
+                    headers: {
+                        'user-agent': req.headers['user-agent'],
+                        host: req.headers.host,
+                    },
+                    body: req.body,
+                    params: req.params,
+                    query: req.query,
+                },
+                res: {
```

Comment:
we may want to log some additional data about our responses to be able to track what happens afterward @BMillman19

Reply:
chatted with @BMillman19 , we will be interested in responses when we do the /quote endpoint, but not for this one. thanks for flagging!